.circleci/config.yml

```diff
@@ -42,6 +41,23 @@ jobs:
           - v1-dependencies-
       - run: yarn && yarn build:bundle && yarn build:pages
 
+  publish:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Write NPM Token to ~/.npmrc
+          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - run:
+          name: Install dot-json package
+          command: npm install dot-json
+      - run:
+          name: Write version to package.json
+          command: $(yarn bin)/dot-json package.json version ${CIRCLE_TAG:1}
```

Comment:
Is this actually being written in the `package.json` found in the repo?

Reply:
yes, and then it's thrown away, version in package.json isn't really needed except when running npm publish

Reply:
if we don't like it, then we can switch to https://github.com/semantic-release/semantic-release or just trigger it by changing version in package.json, but that would leave us without proper changelog on the other hand

Reply:
or I can just commit changes in package.json after publish of course 🙂

Reply:
I think writing changes after publish would be the best approach :)

Reply:
should be updated